Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/playwright
+
+RUN apt update
+RUN apt install python3-pip -y
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+RUN pip install -r requirements.txt
+
+# tricks for pytube : https://github.com/elebumm/RedditVideoMakerBot/issues/142 
```

Comment:
This is not necessary as pytube is no longer a dependency

Reply:
oh I see, I will rectify that